views/home.hbs

```diff
@@ -0,0 +1,23 @@
+<div id="addon">
+    <img class="logo" src="{{manifest.logo}}">
```

Comment:
This needs a maximum width, i can set a 4k logo, and it will show it at that size